internal/cli/cmd/kubeblocks/config.go

```diff
@@ -135,7 +135,7 @@ func NewDescribeConfigCmd(f cmdutil.Factory, streams genericclioptions.IOStreams
 			util.CheckErr(describeConfig(o, output, getHelmValues))
```

Comment:
expected newly add sub-cmd `compare` code.

Reply:
The `compare` doc is missed, the code already is in the main branch.